### DISCUSSION TITLE
Pictarize examples and video autoplay

### DISCUSSION THREAD

**Discussion by legadoandalusi** [hiukim/mind-ar-js#62] [Q&A]
Hi @hiukim and the whole community, first of all, thank you for this magnificent project, and I apologize for my English, I must use Google Translate.
I am trying to create a simple project for my organization (it is a cultural non-profit organization) and I have seen the models that are included in Pictarize, they are fantastic, but I need to host the project on the company server. One of the things that Picturize has is that it allows you to use videos with chroma key, and also allows the videos to be played automatically even on IOS devices.
Is it possible to do this with MindArt? I'm testing with the examples, also combining a-frame functions but I can't get it to work, nor is there anything about it in the documentation. My programming level is very low, I do things by trial error.
Any suggestion?
Thanks in advance

**Comment by brandon-verastegui** [comment]
Hi, yeah I would like to know a dolution for making viddos play through a-frame three.js using mindar
